src/scripts/move.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+// import
+const {
+  getNotEmptyFields,
+  getNotEmptyFieldsCoords,
+  checkBetween,
+  randomField,
+  checkForMerge,
+  checkForMove,
+  setNewHighestScore,
+} = require('./utils');
+
+const {
+  totalCountOfCells,
+  gameScore,
+  allCells,
+  highestScore,
+  values,
+  messageDuringTheGame,
+  messageAfterLose,
+  winCondition,
+} = require('./variables');
+
+function mainListenerRemover() {
+  document.body.removeEventListener('keyup', arrowsPlayHandler);
+}
+
+function createItemInRandomEmptyField() {
+  const emptyFields = [];
+
+  for (const cell of allCells) {
+    if (cell.classList.length === 1) {
+      emptyFields.push(cell);
+    }
+  }
+
+  // create new item:
+  if (emptyFields.length > 0) {
+    const randomIndex = randomField(0, emptyFields.length - 1);
+    const randomValue = randomField(0, values.length - 1);
+
+    emptyFields[randomIndex].style.opacity = '0';
+    emptyFields[randomIndex].style.transition = 'opacity 0.2s';
+
+    emptyFields[randomIndex]
+      .classList.add('field-cell--' + values[randomValue]);
+
+    emptyFields[randomIndex]
+      .innerText = values[randomValue];
+
+    setTimeout(() => {
+      emptyFields[randomIndex].style.opacity = '1';
+      emptyFields[randomIndex].removeAttribute('style');
+    }, 100);
+
+  // if its impossible to create a new item:
+  } else {
+    messageAfterLose.classList.remove('hidden');
+    messageDuringTheGame.classList.add('hidden');
+
+    setNewHighestScore(gameScore, highestScore);
+
+    // mainListenerRemover();
+    document.body.removeEventListener('keyup', arrowsPlayHandler);
+  }
+}
+
+// Main movement Handler:
+function arrowsPlayHandler(e) {
+  switch (e.key) {
+    case 'ArrowDown':
+      mainMove({
+        mainAxis: 0,
+        staticAxis: 1,
+        revert: false,
+        shouldReverse: true,
+      });
+      break;
+
+    case 'ArrowUp':
+      mainMove({
+        mainAxis: 0,
+        staticAxis: 1,
+        revert: true,
+        shouldReverse: false,
+      });
+      break;
+
+    case 'ArrowLeft':
+      mainMove({
+        mainAxis: 1,
+        staticAxis: 0,
+        revert: true,
+        shouldReverse: false,
```

Comment:
Btw, you can skip this field, since it has false value, logic in `if` will work only if it has true value